accel-pppd/radius/radius.c

```diff
@@ -302,6 +302,17 @@ int rad_proc_attrs(struct rad_req_t *req)
 					break;
 			}
 			continue;
+		} else if (attr->vendor && attr->vendor->id == Vendor_Accel_PPP) {
```

Comment:
better to move "if (vendor->id == VENDOR_Accel_PPP)... " down below under "if (attr->vendor)..." block